motoman_driver/src/industrial_robot_client/joint_trajectory_action.cpp

```diff
@@ -532,18 +554,19 @@ void JointTrajectoryAction::controllerStateCB(
     return;
   }
 
-  if (!industrial_utils::isSimilar(all_joint_names_, msg->joint_names))
+  // joint torso_joint_b2 is not in msg->joint_names, so this code is commented out.
```

Comment:
@qqfly: the driver is also used for non-dual-arm robots. This logic will not work there, as the 'shared joint' is a peculiarity of how dual-arm robots are configured.